src/main/resources/com/github/checkstyle/regression/configuration/base_config.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<module name="Checker">
+  <property name="charset" value="UTF-8"/>
+  <!-- do not change severity to 'error', as that will hide errors caused by exceptions -->
+  <property name="severity" value="warning"/>
+  <!-- haltOnException is required for exception fixes and reporting of all exceptions -->
+  <property name="haltOnException" value="false"/>
+  <!-- BeforeExecutionFileFilters is required for sources of java9 -->
+  <module name="BeforeExecutionExclusionFileFilter">
+    <property name="fileNamePattern" value="module\-info\.java$"/>
+  </module>
+
+  <!-- Contents below are generated by regression-tool. -->
+  <module name="TreeWalker"/></module>
```

Comment:
We shouldn't care about generated files as they are out of our control. I'm suprised that it even kept the comments.
Let's only concern ourselves with our files for now.